Lib/Conf.py

```diff
@@ -150,4 +150,14 @@ def getLogo():
   return getCSValue("Logo","/static/core/img/icons/system/_logo_waiting.gif")
 
 def getBackgroud():
-  return getCSValue("BackgroundImage","/static/core/img/wallpapers/dirac_background_6.png")
\ No newline at end of file
+  return getCSValue("BackgroundImage","/static/core/img/wallpapers/dirac_background_6.png")
+
+def getWelcome():
+  res = {}
+  res['show'] = getCSValue("WelcomePage/show", "False")
+  res['style'] = getCSValue("WelcomePage/style", "")
+  res['title'] = getCSValue("WelcomePage/title", "")
+  res['text'] = getCSValue("WelcomePage/text", "")
```

Comment:
Putting large text into the CS is not very practical. I suggest that the returned string is checked if it is a name of existing file (e.g. /opt/dirac/etc/welcome.html) and if it is, read the text from the file.